web_edit_user_filter/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Onestein
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Edit User Filters",
+    "category": "Extra Tools",
+    "version": "14.0.1.0.1",
+    "author": "Onestein, Odoo Community Association (OCA),Level Prime Srl",
```

Comment:
OCA must be kept at the end

Reply:
@simahawk Is it only to keep OCA at the end?

Reply:
y